worlds/shivers/__init__.py

```diff
@@ -202,6 +202,12 @@ def set_lobby_access_keys(items: Dict[str, int]):
             self.options.non_local_items.value |= {name for name, data in item_table.items() if
                                                    data.type in [ItemType.POT, ItemType.POT_COMPLETE]}
 
+        # Ixupi captures priority locations:
+        if self.options.ixupi_captures_priority == 1:
```

Comment:
You can remove the `== 1`
```suggestion
        if self.options.ixupi_captures_priority:
```